frontend/src/features/properties/map/research/detail/PropertyResearchView.tsx

```diff
@@ -0,0 +1,123 @@
+import axios, { AxiosError } from 'axios';
+import { usePropertyDetails } from 'features/mapSideBar/hooks/usePropertyDetails';
+import {
+  InventoryTabNames,
+  InventoryTabs,
+  TabInventoryView,
+} from 'features/mapSideBar/tabs/InventoryTabs';
+import LtsaTabView from 'features/mapSideBar/tabs/ltsa/LtsaTabView';
+import { PropertyDetailsTabView } from 'features/mapSideBar/tabs/propertyDetails/PropertyDetailsTabView';
+import PropertyResearchTabView from 'features/mapSideBar/tabs/propertyResearch/PropertyResearchTabView';
+import useIsMounted from 'hooks/useIsMounted';
+import { useLtsa } from 'hooks/useLtsa';
+import { useProperties } from 'hooks/useProperties';
+import { IApiError } from 'interfaces/IApiError';
+import { IPropertyApiModel } from 'interfaces/IPropertyApiModel';
+import { LtsaOrders } from 'interfaces/ltsaModels';
+import { Api_ResearchFileProperty } from 'models/api/ResearchFile';
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { pidFormatter } from 'utils';
+
+export interface IPropertyResearchViewProps {
+  researchFileProperty: Api_ResearchFileProperty;
+  setEditMode: (isEditing: boolean) => void;
+}
+
+const PropertyResearchView: React.FunctionComponent<IPropertyResearchViewProps> = props => {
```

Comment:
This seems more like a container then a view to me, lots of logic and almost no display

Reply:
alternatively we could revisit this:

Naming
Top level react components accessible via the main AppRouter should be postfixed with Page.
A Page is composed of one or more Containers, that provide logic for child Views.
A View is ideally a stateless via only react component. All logic should be provided by a Container.
A View is comprised of multiple components, which do not have an enforced naming pattern.
Note that there may be multiple Containers in a Page, and that a Container may contain other nested Containers.
Note that some components will not implement the previous pattern, but all new code should.

(from https://confluence.th.gov.bc.ca/display/PIMS/Best+Practices)

Reply:
Fair point, I think I just named view since it was at the same level of the other files